test/plugin/scan-test.js

```diff
@@ -65,6 +65,7 @@ function test(executable, core, t) {
     }
     t.ok(found, 'Zlib should be in findjsinstances');
 
+    sess.send("target modules dump symtab");
```

Comment:
Can you separate this out into another test case? (That is, put this in the next `sess.linesUntil` callback, call another `sess.send('version');` there, and move the tests below into a new `sess.linesUntil(versionMark...)` session.) That way it would be easier to debug without mixing the `target modules dump symtab` output in the middle.

(also nit: single quotes)